README.md

```diff
@@ -1,17 +1,17 @@
 [![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-# New Relic Koa Instrumentation [![koa Instrumentation CI][1]][2]
+# New Relic Koa instrumentation [![koa instrumentation CI][1]][2]
```

Comment:
totally fine but just calling out the second instance here is actually the proper title of the workflow in the GitHub actions UI.